packages/cf-deploy-config-writer/src/translations/cf-deploy-config-writer.i18n.json

```
@@ -14,9 +14,12 @@
         "cleanupModules": "Cleanup modules.",
         "creatingMta": "Creating MTA using {{- cdsParams }}...",
         "html5AppAdded": "HTML5 application added to MTA configuration {{- appName}}",
+        "html5AlreadyExists": "HTML5 application {{- appName}} already exists in the MTA configuration.",
         "addXsuaaService": "Adding XSUAA service to MTA configuration.",
         "mtaReadWithPrefix": "Read mta.yaml with prefix {{- prefix }}.",
-        "errorReadingMta": "Error reading mta.yaml file {{- error }}."
+        "errorReadingMta": "Error reading mta.yaml file {{- error }}.",
```

```suggestion
        "errorReadingMta": "An error when reading the `mta.yaml` file. {{- error }}.",
```
What should users do to try to fix this error? View the logs?